velox/exec/LocalPartition.cpp

```diff
@@ -77,19 +77,18 @@ void LocalExchangeQueue::noMoreProducers() {
 
 BlockingReason LocalExchangeQueue::enqueue(
     RowVectorPtr input,
+    int64_t size,
```

Comment:
nit: perhaps, rename to inputBytes or similar to make it clear that this is size in bytes, not number of rows